src/graphql/schema.gql

```diff
@@ -521,6 +533,8 @@ type User {
   name: String
   about: String
   avatar: String
+  vote_count: Int
+  proposal_count: Int
```

Comment:
Maybe we could add last_vote also?

Reply:
We could, but it's not part of SX api, and will not be used anywhere (for now)

Reply:
I think better to add it anyway to know the last voted timestamp of a user, but your decision, not too important